submissions/Mifaresss/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Popup | Homework Kottans</title>
+	<link href="https://fonts.googleapis.com/css?family=Noto+Sans:regular,700&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+	<header class="header">
+		<div class="wrapper">
+			<div class="header__container">
+				<a href="#" class="header__logo">PopUp</a>
+				<nav class="nav">
+					<label for="popap-icon" class="label">
+						<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="35" height="35" viewBox="0 0 16 16">
+							<path fill="#212121" d="M0 7h7v-7h-7v7zM1 1h5v5h-5v-5z"></path>
+							<path fill="#212121" d="M9 0v7h7v-7h-7zM15 6h-5v-5h5v5z"></path>
+							<path fill="#212121" d="M0 16h7v-7h-7v7zM1 10h5v5h-5v-5z"></path>
+							<path fill="#212121" d="M9 16h7v-7h-7v7zM10 10h5v5h-5v-5z"></path>
+						</svg>
+					</label>
+					<input type="checkbox" id="popap-icon" class="popup-button">
+					<div class="popup">
+						<ul class="popup__list">
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/01.svg" alt="icon1">Спидометр</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/02.svg" alt="icon2">Будильник</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/03.svg" alt="icon3">Якорь</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/04.svg" alt="icon4">Часы</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/05.svg" alt="icon5">Фиг знает</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/06.svg" alt="icon6">Wi-Fi</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/07.svg" alt="icon7">"Стоп"</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/08.svg" alt="icon8">Энергия</a></li>
+							<li class="popup__item"><a class="popup__link" href="#"><img class="popup__img" src="./img/09.svg" alt="icon9">Батарея</a></li>
+							<li class="popup__item">
+								<label class="sub-label" for="popup-sub-icon"><span>More</span></label>
```

Comment:
Hide the 'more' button when the second part of the popup is open

Reply:
Okey, will be done

Reply:
Is there a need for improvement here?